bibtasklets/bst_fermilab.py

```diff
@@ -225,6 +252,20 @@ def bst_fermilab():
     write_message('\\rm fermilab-reports-preprints.html')
     write_message('cp %s .' % filename)
 
+    if ERRORS:
+        send_notification_email(ERRORS)
+
+def send_notification_email(ERRORS):
```

Comment:
a function of the same name, but with different arguments is already defined on line 55 above.